oauth2client/contrib/appengine.py

```diff
@@ -210,6 +211,35 @@ def create_scoped_required(self):
     def create_scoped(self, scopes):
         return AppAssertionCredentials(scopes, **self._kwargs)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
```

Comment:
Might we compromise on adding something like "Implements abstract method AppAssertionCredentials.sign_blob" to this doc string? I'm comfortable backpedalling to allowing the copied doc string but there's got to be something informing future users and maintainers about the relationship between this code element and another code element elsewhere.

(If you agree, alter the similar "ditto" copied doc comment elsewhere in this change?)